DO-987 updated the start monit and stop monit tasks in deployment script

Kaali09 opened the pull request:
Added the monit stop, monit start and remove old artifacts tasks in secor, analytics and logstash deployment scripts.

keshavprasadms commented on ansible/lpa_telemetry_backup_deploy.yml:
```diff
@@ -2,6 +2,16 @@
   vars_files:
     - "{{inventory_dir}}/secrets.yml"
+  pre_tasks:
+    - name: Stop the monit
```
@Kaali09 Please add logic to run this block only if monit exists in server. Since adopters may not use monit, this should not be a blocker. But if monit exisits, this block cannot include ignore_errors. If monit fails to start / stop, the deployment should fail. Same for other blocks.

Kaali09 replied:
Its resolved.

Kaali09 commented:
@keshavprasadms Please review the new changes.

Kaali09 commented:
@anandp504 We have updated the secor, analytics-api and logstash deployment script with Stop monit and start monit service.  This addition is to make sure monit should not play any role when deployment happens.

Also, we have added the tasks to remove the old artifatcs file in the server before copy the new artifacts into the server to make sure new code deployed . Please review from your end and suggest if any.

We also working on refactoring the samza-jobs deployment script.
